private/rules/jvm_import.bzl

```diff
@@ -8,13 +8,26 @@
 # [1]: https://github.com/bazelbuild/bazel/issues/4549
 
 load("@rules_java//java:defs.bzl", "JavaInfo")
+load("@rules_license//rules:providers.bzl", "PackageInfo")
+load("//private/lib:coordinates.bzl", "to_external_form", "to_purl", "unpack_coordinates")
+load("//private/lib:urls.bzl", "scheme_and_host")
 load("//settings:stamp_manifest.bzl", "StampManifestProvider")
 
 def _jvm_import_impl(ctx):
-    if len(ctx.files.jars) != 1:
+    if not ctx.attr.jar and not ctx.attr.jars:
```

Comment:
nit: this should probably be an independent change.

Reply:
I'll break it out.

Reply:
https://github.com/bazelbuild/rules_jvm_external/pull/1242